### PR TITLE
feat(header): L3-4227 header changes

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-4227](https://phillipsauctions.atlassian.net/browse/L3-4227)
| Before | After |
| ------ | ----- |
| image | image |

[L3-4230](https://phillipsauctions.atlassian.net/browse/L3-4230)
| Before | After |
| ------ | ----- |
| image | image |

[L3-4193](https://phillipsauctions.atlassian.net/browse/L3-4193)
| Before | After |
| ------ | ----- |
| image | image |
| Before | After |
| ------ | ----- |
| image | image) |

**Summary**

Fixed some spacing, changed Mobile Hero

**Change List (describe the changes made to the files)**

- src/components/Accordion/_accordion.scss: updated mobile hamburger spacing
- src/patterns/HeroBanner/HeroBanner.tsx: changed type of heading
- src/patterns/HeroBanner/_heroBanner.scss: added new/old mobile version
- src/scss/_vars.scss: added new snw
- src/site-furniture/Header/_header.scss: updated logo's size ([here](https://phillipsauctions.atlassian.net/browse/L3-4230))

**Acceptance Test (how to verify the PR)**

- verify that header on seldon is the same as prod, mobile and desktop
- verify that logo is 20px
- verify that padding in mobile header is 20px

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-4227]: https://phillipsauctions.atlassian.net/browse/L3-4227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[L3-4230]: https://phillipsauctions.atlassian.net/browse/L3-4230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[L3-4193]: https://phillipsauctions.atlassian.net/browse/L3-4193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/site-furniture/Header/_header.scss [hunk 1]
```diff
@@ -77,15 +77,15 @@
...
       width: 100%;
     }
 
     @include isHeaderDesktop {
       padding-top: 0;
 
       svg {
-        height: 28px;
+        height: 25.73px; // 25.73px is the height of the logo's container to make logo 20px
         width: 100%;
```
Comment (davidicus) [inline src/site-furniture/Header/_header.scss:89]: Is this needed?

**Review by davidicus — COMMENTED** [review]
Looks good. Just a few minor comments

File: src/site-furniture/Header/_header.scss (hunk 1 above)
Comment (constantinehuzenko) [reply src/site-furniture/Header/_header.scss:89]: the height of svg 
image

the logo itself
image

File: src/patterns/HeroBanner/_heroBanner.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 .#{$px}-hero-banner {
   align-items: center;
-  background: linear-gradient(rgba(0, 0, 0, 60%), rgba(0, 0, 0, 60%)), var(--background);
+  background: linear-gradient(rgba(0, 0, 0, 50%), rgba(0, 0, 0, 50%)), var(--background);
```
[the remark this answers is not in the document]
Comment (davidicus) [reply src/patterns/HeroBanner/_heroBanner.scss:5]: I guess this would just have to be a string variable that we export to be used in the react file.

**Review by davidicus — APPROVED** [review]
We have some discussions left to be had about this gradient prop but I am going to merge for now so we can deliver the header PR.

File: src/patterns/HeroBanner/_heroBanner.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidicus) [reply src/patterns/HeroBanner/_heroBanner.scss]: I was thinking more of having a string variable that we export like our `px` variable that we can use. This way we are not adding an additional prop. 

However, this way does allow us to set a default. Not sure if that value offsets the additional prop but something to consider.

**Pull request merged**